goon-universe/app.js

```diff
@@ -226,7 +226,7 @@ router.route('/scenarios/initialAction')
         }
         else{
         db.getInitActionPageQuestionsAndChoices(scenarioID, function(result){
```

Comment:
The action GET endpoints should return the question and choice IDs that the action PUT endpoint expects.

Reply:
This should be fixed by the latest commit.